enterprise/frontend/src/embedding-sdk/cli/steps/setup-metabase-instance.ts

```diff
@@ -178,11 +178,6 @@ export const setupMetabaseInstance: CliStepMethod = async state => {
 
     spinner.succeed();
 
-    printInfo(
```

Comment:
Moved this message to the end of the step, as it's easy to miss this message after entering the DB connection info.